install_files/ansible-base/usr.sbin.apache2

```diff
@@ -206,6 +206,7 @@
   /var/www/securedrop/static/i/tipbox/tipbox-hed-submit3.png r,
   /var/www/securedrop/static/i/tipbox/tipbox-logo.png r,
   /var/www/securedrop/static/js/journalist.js r,
+  /var/www/securedrop/static/js/libs/jquery-2.1.4.js r,
```

Comment:
Do we need to add this to the AppArmor rules if this is distributed only for legal reasons?